elfo-core/src/supervisor/backoff.rs

```diff
@@ -1,20 +1,20 @@
 use std::time::Duration;
 
+use crate::RestartParams;
 use elfo_utils::time::Instant;
 
-const BACKOFF_STEP: Duration = Duration::from_secs(5);
-const MAX_BACKOFF: Duration = Duration::from_secs(30);
-
 pub(crate) struct Backoff {
```

Comment:
It's not obvious to me now why this struct is placed here, not inside `mod restarting`. Or the module should be named `restart_policy`.

Reply:
moved to the mod and changed the struct name to `RestartBackoff`